Sources/Utility/ArgumentParser.swift

```diff
@@ -193,15 +214,15 @@ public final class PositionalArgument<Kind>: ArgumentProtocol {
 /// A type-erased argument.
 ///
 /// Note: Only used for argument parsing purpose.
-fileprivate final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
+public final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
```

Comment:
Why do you need AnyArgument to be public? See the note, it was only meant to be used by argument parser internally